### PR TITLE
OBDS-78 Added fields to location import

### PR DESCRIPTION
**Pull request by MagdaToczek**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/data/LocationDataService.groovy [hunk 1]
```diff
@@ -109,4 +116,36 @@ class LocationDataService {
...
         return location
     }
+
+    Address createOrUpdateAddress(Map params, @Nullable String addressId) {
+        Address address
+        if (addressId) {
+            address = Address.findById(addressId)
+        }
+
+        if (!address) {
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/data/LocationDataService.groovy:126]: You could just do:
```
if (!address) {
  address = new Address()
}
```
and then do the params assign code once

File: grails-app/services/org/pih/warehouse/data/LocationDataService.groovy [hunk 2]
```diff
@@ -39,6 +41,10 @@ class LocationDataService {
                 command.errors.reject("Row ${index + 1}: '${params.name}' already exists in the system. Choose a unique name")
             }
 
+            if (!params.activeOrInactive || (!(params.activeOrInactive instanceof Boolean) && !("true".equalsIgnoreCase(params.activeOrInactive) || "false".equalsIgnoreCase(params.activeOrInactive)))) {
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/data/LocationDataService.groovy:44]: Call it `active`. How you'd like set `active` field when `active or inactive is true` Is it both active and inactive? :smile:

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/services/org/pih/warehouse/data/LocationDataService.groovy (hunk 1 above)
Comment (MagdaToczek) [reply grails-app/services/org/pih/warehouse/data/LocationDataService.groovy:126]: done

File: grails-app/services/org/pih/warehouse/data/LocationDataService.groovy (hunk 2 above)
Comment (MagdaToczek) [reply grails-app/services/org/pih/warehouse/data/LocationDataService.groovy]: done

File: grails-app/domain/org/pih/warehouse/core/Location.groovy [hunk 3]
```diff
@@ -306,12 +307,19 @@ class Location implements Comparable<Location>, java.io.Serializable {
...
 
     static PROPERTIES = [
-            "name"          : "name",
-            "locationNumber": "locationNumber",
-            "locationType"  : "locationType.name",
-            "locationGroup" : "locationGroup.name",
-            "parentLocation": "parentLocation.name",
-            "organization"  : "organization.name"
+            "name"            : "name",
+            "activeOrInactive": "active",
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/core/Location.groovy:311]: @MagdaToczek you forgot about this one

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**